server/portal/apps/projects/tasks.py

```diff
@@ -0,0 +1,79 @@
+"""
+.. :module:: portal.apps.projects.tasks
+   :synopsis: Tasks pertaining to projects
+"""
+from __future__ import unicode_literals, absolute_import
+import logging
+from celery import shared_task
+from django.contrib.auth import get_user_model
+from portal.apps.projects.models import Project
+
+# pylint: disable=invalid-name
+logger = logging.getLogger(__name__)
+# pylint: enable=invalid-name
+
+
+@shared_task(
```

Comment:
Do we have any guarantees about what happens if the celery task doesn't happen immediately and the UI isn't updated immediately?